Ports/radare2/package.sh

```diff
@@ -2,11 +2,10 @@
 port=radare2
```

Comment:
This commit title is misleading, the radare2 port is already version 5.7.0

Reply:
Yes, I was unsure about that. I changed it to state the most important change now.